### ISSUE TITLE
some items are not consistently available in filter-selectors

### ISSUE THREAD

**Issue by Calibretto22** [raiguard/Factorio-EditorExtensions#10]
I don't know how the availability of the items is controlled in the definitions. 
But several inconsistencies have come to my attention: 

**"Infinity pipes":** 
- infinity-filters work as expected
![image](https://user-images.githubusercontent.com/29695655/80311717-8828fb80-87e1-11ea-96d7-7353a3e58018.png)

- not available in selection of slot-filters
![image](https://user-images.githubusercontent.com/29695655/80311489-3cc21d80-87e0-11ea-867e-f3cfbd912d44.png)

- not available in selection of logistic-filters 
![image](https://user-images.githubusercontent.com/29695655/80311482-274cf380-87e0-11ea-838a-e5aa7169c106.png)

- not available in selection of request-filters
![image](https://user-images.githubusercontent.com/29695655/80311452-f79deb80-87df-11ea-9b71-7b6431b567c9.png)

- not available in selection of blueprint-icons
![image](https://user-images.githubusercontent.com/29695655/80311384-ac83d880-87df-11ea-8c44-51df6dd7f3d1.png)

- not available in selection of deconstruction-filters
![image](https://user-images.githubusercontent.com/29695655/80311578-c114a080-87e0-11ea-9df0-81ae17bf874a.png)

the last picture seems to show that old prototypes/items (?) are still present or not properly migrated (i.e. multiple inifity loaders & accumulators, blue infinity pipe & pink infinity pipe). 

drag and drop of items to set a filter works fine.

**Comment by raiguard** [comment]
Thanks for the report, I have fixed the infinity pipe being missing.

As for the duplicate items, there are two reasons for this: The infinity pipe, infinity chest, and infinity heat pipe are duplicates of vanilla entities/items. The vanilla equivalent to the infinity pipe is tinted dark blue, while EE's infinity pipe is pink. Same goes for the infinity chest - the EE chest has a slightly different color from the vanilla chest, to let you distinguish them. The heat pipe is just a reskinned heat interface.

The infinity accumulator and infinity loader are duplicated in entity filter menus because there are actually several entities for them. The infinity loader has a "dummy" entity that you place, which is immediately replaced with a "logic" entity that you actually interact with. The accumulator has five different versions depending on the mode and priority, because those can't be changed at runtime.

So, the only actual bug here is the infinity pipe being missing, and I have fixed that. Let me know if you have any other questions!
